components/navbar/sidebar.snip.html

```diff
@@ -25,15 +25,15 @@
     <ul class="list-reset m0 p0 ampstart-label">
       {{#nav}}
         {{#link}}
-          <li class="ampstart-nav-item"><a class="ampstart-nav-link" href="{{{url}}}">{{text}}</a></li>
+          <li class="ampstart-nav-item {{classes}}"><a {{#aria-label}}aria-label="{{.}}"{{/aria-label}} class="ampstart-nav-link" href="{{{url}}}">{{{text}}}</a></li>
```

Comment:
Allow classes to be applied from the JSON file. Adding this removes a significant amount of CSS from our custom css files.